generated/tests/server/routes/members-only-test.js

```diff
@@ -45,10 +45,8 @@ describe('Members Route', function () {
 			password: 'shoopdawoop'
 		};
 
-		beforeEach('Create a user', function (done) {
-			return User.create(userInfo).then(function (user) {
-                done();
-            }).catch(done);
+		beforeEach('Create a user', function () {
+			return User.create(userInfo);
```

Comment:
FYI this is because Mocha 3 throws an error when the user (incorrectly) attempts to both return a promise *and* call `done`.